napari/_qt/qt_range_slider.py

```diff
@@ -50,10 +74,12 @@ def __init__(
         self.setHandleBorderColor(QtGui.QColor(200, 200, 200))
 
         self.setEnabled(True)
+        if not parent:
+            self.setGeometry(200, 200, 200, 20)
```

Comment:
Is the identical call below (line 85) still needed, then?

Reply:
(that's a thumbs down for, "nope" ... 😊)